src/app/model/tollbooth-factory.ts

```diff
@@ -42,5 +42,13 @@ export class TollboothFactory {
       tollbooth.beginGeneration();
       tollbooths.push(tollbooth);
     }
+
+    this.renderTargets(lanes, top);
```

Comment:
Consider calling this at world level, but keep it at the same level as rendering tollbooths regardless of where that is